lldb/test/API/tools/lldb-dap/attach/TestDAP_attachByPortNum.py

```diff
@@ -0,0 +1,137 @@
+"""
+Test lldb-dap "port" configuration to "attach" request
+"""
+
+
+import dap_server
+from lldbsuite.test.decorators import *
+from lldbsuite.test.lldbtest import *
+from lldbsuite.test import lldbutil
+import lldbdap_testcase
+import os
+import shutil
+import subprocess
+import tempfile
+import threading
+import time
+import sys
+
+
+class TestDAP_attachByPortNum(lldbdap_testcase.DAPTestCaseBase):
+    def runTargetProgramOnPort(self, port=None, program=None):
+        server_tool = "lldb-server"
```

Comment:
lldb-server is available on linux, and gdbserver on mac, so you need to look for both depending on the situation.

@JDevlieghere , is there an SB API that we could use to get the path to the gdb-server that LLDB uses?

Reply:
I think "**lldbgdbserverutils.get_debugserver_exe()**" this function would help us to get gdbserver path.

Reply:
that's a good idea!